content/en/docs/IV/modules/hosp/emar.md

```diff
@@ -19,6 +19,8 @@ Records in this table are populated by bedside nursing staff scanning barcodes a
 * *prescriptions* on `pharmacy_id`
 * *poe* on `poe_id`
 
+Note: The eMAR system was implemented during the year 2011-2013. Therefore, patients do not have records in emar table might because the information wasn’t available. 
```

Comment:
Please could you follow the style of other pages and add this in the "Important considerations" section?

Instead of: "The eMAR system was implemented during the year 2011-2013. Therefore, patients do not have records in emar table might because the information wasn’t available."

perhaps something like:

"The eMAR system was implemented during between the 2011-2013. As a result, eMAR data is not available for all patients."

Reply:
Thanks for reviewing! I have updated the suggested changes.

Reply:
Sorry, typo in the suggested text: either "during" or "between" but not both! ...

"The eMAR system was implemented between 2011-2013. As a result, eMAR data is not available for all patients."

Reply:
@tompollard I have updated the two .md files.